salt/_runners/metalk8s_saltutil.py

```diff
@@ -21,7 +21,12 @@ def sync_auth(saltenv='base', extmod_whitelist=None, extmod_blacklist=None):
 def wait_minions(tgt='*', retry=10):
     client = salt.client.get_local_client(__opts__['conf_file'])
 
-    minions = client.cmd(tgt, 'test.ping', timeout=2)
+    try:
+        minions = client.cmd(tgt, 'test.ping', timeout=2)
+    except Exception as exc:
+        log.warning('Unable to run "test.ping" on "%s": "%s"', tgt, exc)
```

Comment:
Simply a `warning`? Why not `log.exception` to get the traceback? Do we know exactly what kind of error can trigger this, and know that we can safely ignore them / hide them from the user?

Reply:
Ok, I don't really care I use `log.exception`